src/modules/Layout/Row/Row.scss

```diff
@@ -1,5 +1,20 @@
 .j-row {
-  display: flex;
-  flex-direction: row;
-  justify-content: space-between;
+    display: flex;
+    width: 100%;
```

Comment:
can you try a combination of `flex-grow` and, `flex-shrink` and see if you still need the 100%